guides/common/modules/con_managing-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,32 @@
+[id="managing_multiple_content_view_environments_{context}""]
+= Managing multiple content view environments
+
+You can assign hosts and activation keys to one or more content view environments instead of assigning lifecycle environments and content views separately.
+A content view environment combines a specific lifecycle environment with a content view, streamlining the assignment process.
+When you assign a host to multiple content view environments, the host gains access to the combined repositories from all its associated content view environments.
+
+The `allow_multiple_content_views` setting controls whether hosts and activation keys can be assigned to multiple content view environments.
+By default, this feature is disabled, limiting assignments to a single content view environment.
+Enabling the setting allows you to assign multiple content view environments to both hosts and activation keys.
+For more information on general settings, see {AdministeringDocURL}general_settings_information_[General settings] in _{AdministeringDocTitle}_.
+
+When `allow_multiple_content_views` is enabled:
+
+* Hosts and activation keys can be assigned to one or more content view environments.
+* Hosts registered with multiple activation keys handle content view environment assignments based on the key order and settings configuration.
+
+When `allow_multiple_content_views` is disabled:
```

Comment:
Does this describe what happens when a user enables mult. CVs and then disables them? If this is the case, you need to mention it.

Reply:
Yes.